python_modules/libraries/dagster-powerbi/dagster_powerbi_tests/pending_repo.py

```diff
@@ -28,3 +31,18 @@ def my_materializable_asset():
         pbi_defs,
     ).get_inner_repository(),
 )
+
+
+@definitions
+def source_metadata_defs(context: DefinitionsLoadContext) -> Definitions:
+    pbi_defs = load_powerbi_defs(
+        context,
+        api_token=fake_token,
+        workspace_id="a2122b8f-d7e1-42e8-be2b-a5e636ca3221",
+        enable_refresh_semantic_models=False,
+    )
+
+    return Definitions.merge(
+        Definitions(assets=[my_materializable_asset], jobs=[define_asset_job("all_asset_job")]),
+        pbi_defs,
+    )
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +36 to +49]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Much better<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24403?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->